src/BwdNotation.mli

```diff
@@ -1,2 +1,2 @@
-include module type of BwdNoLabels.Notation
+include module type of BwdNoLabels.Infix
```

Comment:
Shouldn't this file get renamed?

Reply:
`BwdNotation` exists only for backward compatibility---it would be removed instead if not for that, unless you want `BwdInfix` in addition to `Bwd.Infix` (which was not part of my plan).

Reply:
I see. So the reason to not make the breaking change is to prevent a library-storm?

Reply:
Practically I think we could just break the interface now because no one else is using it (is it still the case?), but I kind of want to honor our promise that the interface is sort of stable.

Reply:
ok!